allure-pytest/examples/display_name/display_name.rst

```diff
@@ -17,3 +17,8 @@ Titles support placeholders for arguments.
     ... @pytest.mark.parametrize('param', [False])
     ... def test_display_name_template(param):
     ...     assert param
+
+    >>> @allure.title("A some test title with ParameterSet id {param_id}")
```

Comment:
```suggestion
    >>> @allure.title("A test title with ParameterSet id {param_id}")
```

Reply:
Sure. Didn't notice it. Just copied it from the example few lines above :)